docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -160,32 +88,63 @@ PacketFence supports the following operating systems on the x86_64 architecture:
+Installation
+------------
 
-[options="compact"]
-* Web server (httpd)
-* DHCP server (dhcpd)
-* FreeRADIUS server (radiusd)
-* Firewall (iptables)
+This section will guide you through the installation of PacketFence from the Zero Effort NAC (ZEN) appliance and from the standard repository of packages we provide - which can be used to install PacketFence on top of a vanilla GNU/Linux installation.
 
-*Make sure that all the other services are automatically started by your operating system!*
+Installing PacketFence from the ZEN
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Installation
-------------
+The ZEN (Zero Effort NAC) edition of PacketFence allows you to rapidly get PacketFence running in your network environment. It consists of a fully installed and preconfigured version of PacketFence distributed as a virtual appliance. It can be deployed on VMware ESX/ESXi, Microsoft Hyper-V and other products. This section covers the deployment of the virtual appliance on VMware-based products. We are not supporting any Xen-based hypervisors yet.
+
+Virtual Machine
+^^^^^^^^^^^^^^^
 
-This section will guide you through the installation of PacketFence together with its dependencies.
+This setup has been tested using VMWare ESXi, Fusion and Workstation products with 8 GB of RAM dedicated to the virtual machine. It might work using other VMWare products.  To properly run the PacketFence virtual appliance, need a CPU that supports long mode. In other words, you need to have a 64-bit capable CPU on your host. PacketFence ZEN comes in a pre-built virtual disk (OVF). If you are using an ESX type hypervisor, you need to import the OVF using vSphere Client (or vCenter).
 
-OS Installation
-~~~~~~~~~~~~~~~
+The virtual appliance passwords are:
+
+.Management (SSH/Console) and MySQL
+* Login: root
+* Password: p@ck3tf3nc3
+
+.Captive Portal / 802.1X Registration User
+* Login: demouser
+* Password: demouser
+
+
+Import to ESX
+^^^^^^^^^^^^^
+
+Make sure that there is only one virtual network card created, and also make sure that your vEthernet is connected to a virtual switch (vSwitch). You will need to create a “TRUNK” profile to allow all VLAN tags (usually VLAN 4095), and assign the profile to the PacketFence ZEN VM vEthernet.
+
+Import to VMWare Player/Workstation for Linux
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Newer version of VMWare Player handles the VLAN trunking a lot better.  Having that said, we can use a single interface on the VM.  So, you need to ensure that your VM host is plugged into a physical trunk port with VLAN 1,2,3,5,10 and 200 as the allowed VLAN. These VLANs will be used later in configuration examples.
+
+IMPORTANT: Please refer to the Annex 1 if you have troubles using Workstation with only one trunked interface.  We tested most with VMWare Player for Linux. We cannot support VMWare Fusion anymore, there are some issues when we need to do routing through Mac OS X VLAN interfaces.
```

Comment:
Annex 1 seems to be missing but anyway the trunk might not be necessary anymore so if that is the case, it could be removed